internal/operators/cnv/manifest.go

```diff
@@ -6,34 +6,34 @@ import (
 )
```

Comment:
@jordigilh pls change the import package from `html/template` to `text/template`

Reply:
will do